how-to/2_Geometry/CreateCustomGeometry.ipynb

```diff
@@ -0,0 +1,410 @@
+    "#  Copyright 2021 - 2024 The University of Manchester\n",
+    "#\n",
+    "#  Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "#  you may not use this file except in compliance with the License.\n",
+    "#  You may obtain a copy of the License at\n",
+    "#\n",
+    "#      http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "#  Unless required by applicable law or agreed to in writing, software\n",
+    "#  distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "#  See the License for the specific language governing permissions and\n",
+    "#  limitations under the License.\n",
+    "#\n",
+    "#   Authored by:    Mariam Demir (UKRI-STFC)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create AcquisitionGeometry Using Dataset Metadata\n",
+    "##### This example shows how to use the dataset's metadata to manually create and visualise the AcquisitionGeometry, and the AcquisitionData "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from cil.io import TIFFStackReader\n",
+    "from cil.framework import AcquisitionGeometry, AcquisitionData\n",
+    "from cil.utilities.display import show_geometry\n",
+    "from cil.utilities.display import show2D\n",
+    "from zipfile import ZipFile\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download and extract the file `SparseBeads_B12_L1` from Zenodo [here](https://zenodo.org/records/290117/files/SparseBeads_B12_L1.zip)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "--2024-10-19 16:39:46--  https://zenodo.org/records/290117/files/SparseBeads_B12_L1.zip\n",
+      "Resolving zenodo.org (zenodo.org)... 188.184.98.238, 188.184.103.159, 188.185.79.172, ...\n",
+      "Connecting to zenodo.org (zenodo.org)|188.184.98.238|:443... connected.\n",
+      "HTTP request sent, awaiting response... 200 OK\n",
+      "Length: 20588556 (20M) [application/octet-stream]\n",
+      "Saving to: ‘/home/efv97572/datasets/SparseBeads_B12_L1.zip.1’\n",
+      "\n",
+      "SparseBeads_B12_L1. 100%[===================>]  19.63M  52.6MB/s    in 0.4s    \n",
+      "\n",
+      "2024-10-19 16:39:47 (52.6 MB/s) - ‘/home/efv97572/datasets/SparseBeads_B12_L1.zip.1’ saved [20588556/20588556]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "!wget -P ~/datasets https://zenodo.org/records/290117/files/SparseBeads_B12_L1.zip\n",
+    "\n",
+    "z = ZipFile(os.path.expanduser(\"~/datasets/SparseBeads_B12_L1.zip\"))\n",
+    "z.extractall(os.path.expanduser(\"~/datasets\"))"
```

Comment:
The nicest way to do this would be to add the spare beads dataset to our dataexample class. But we should discuss this in the meeting later today.